src/index.html

```diff
@@ -214,52 +214,74 @@
                     <h2>Submit</h2>
                     <h3>European Variation Archive submissions</h3>

```

Comment:
Isn't this header kind of redundant, given that it is in the "Submit data" section?